crates/libcontainer/src/process/container_init_process.rs

```diff
@@ -417,11 +413,7 @@ pub fn container_init_process(
 
     if proc.args().is_some() {
         ExecutorManager::exec(spec)?;
-        if fifo_fd != 0 {
-            let f = &mut unsafe { File::from_raw_fd(fifo_fd) };
-            // TODO: impl
-            write!(f, "1")?;
-        }
```

Comment:
Ideally this will never execute, as exec call should replace the process image by the exec command. I think the return type of exec should be replaced from `Result<()>` to `Result<!>` as per https://doc.rust-lang.org/std/primitive.never.html#infinite-loops